stable/kong/README.md

```diff
@@ -539,14 +548,14 @@ Following changes have no end user visible effects:
 
 #### Fixes
 
-- Do not remove whitespace between documents when rendering
+- Do not remove white-space between documents when rendering
```

Comment:
Space instead of hyphen, see https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/white-space